src/main/java/net/alpenblock/bungeeperms/BungeePerms.java

```diff
@@ -7,6 +7,7 @@
 import net.alpenblock.bungeeperms.platform.NetworkNotifier;
 import net.alpenblock.bungeeperms.platform.PlatformPlugin;
 import net.alpenblock.bungeeperms.platform.PluginMessageSender;
+import org.bukkit.configuration.file.FileConfiguration;

```

Comment:
this import is not working in this class ... the BungeePerms class is intended to not depend on bukkit or bungee api